common/lib/capa/capa/tests/test_hint_functionality.py

```diff
@@ -52,7 +52,7 @@ def test_tracking_log(self):
         self.get_hint(u'1_3_1', u'Blue')
         self.problem.capa_module.runtime.track_function.assert_called_with(
             'edx.problem.hint.feedback_displayed',
-            {'module_id': 'i4x://Foo/bar/mock/abc',
+            {'module_id': u'i4x://Foo/bar/mock/abc',
```

Comment:
Are these fixes to assert_called_with parameters consequential to any other change in this PR?  I'm just curious because it doesn't look like you changed anything that would have caused what was previously a py2 str to become a py2 unicode.

Reply:
The old mock in `capa/tests/helpers.py` returned a byte string, and the new one returns a unicode string (to better match the method it's mocking).  The assertions may still have passed without these changes, but the new expected values are more accurate.

Reply:
Oh, I see that now.